pincer/objects/guild/channel.py

```diff
@@ -779,6 +779,57 @@ async def fetch_message(self, message_id: int) -> UserMessage:
             await self._http.get(f"channels/{self.id}/messages/{message_id}")
         )
 
+    async def history(
+        self, limit: int = 50,
+        before: Optional[Union[int, Snowflake]] = None,
+        after: Optional[Union[int, Snowflake]] = None,
+        around: Optional[Union[int, Snowflake]] = None,
+    ) -> AsyncIterator[UserMessage]:
+        """|coro|
+        Returns a list of messages in this channel.
+
+        Parameters
+        ----------
+        around
+            The message ID to look around.
+        after
+            The message ID to look after.
+        before
+            The message ID to look before.
+        limit
+            The maximum number of messages to return.
```

Comment:
Add types in the docstring
```suggestion
        around : Optional[Union[:class:`int`, :class:`Snowflake`]]
            The message ID to look around.
        after : Optional[Union[:class:`int`, :class:`Snowflake`]]
            The message ID to look after.
        before : Optional[Union[:class:`int`, :class:`Snowflake`]]
            The message ID to look before.
        limit : Optional[Union[:class:`int`, :class:`Snowflake`]]
            The maximum number of messages to return.
```